example/android/shared/build.gradle.kts

```diff
@@ -23,9 +20,12 @@ kotlin {
     ).forEach {
         it.binaries.framework {
             baseName = "shared"
+            isStatic = true
```

Comment:
Why is this change needed?

Reply:
I think I added this when I was debugging something, just to take the dynamic linking out of the equation. I think we can remove it again. I'll validate the removal and update the PR tomorrow.

Reply:
Validated and removed in https://github.com/voize-gmbh/reakt-native-toolkit/pull/53/commits/edc61c9f85afd01c351bf23049615096e60c5977. Also added https://github.com/voize-gmbh/reakt-native-toolkit/pull/53/commits/d5c5985270d6710537354f88f17a213e826ca9f1.